pkg/cli/tsdump.go

```diff
@@ -110,7 +110,7 @@ will then convert it to the --format requested in the current invocation.
 		case tsDumpDatadog:
 			w = makeDatadogWriter(
 				ctx,
-				"https://api.datadoghq.com/api/v2/series",
```

Comment:
This can be taken as a CLI flag with the default value being `us5`. This is what I've done for the debug zip upload as well. I've taken the site as a flag and inferred the relevant domain names based on that.

This may not change in the near future, but it will be good to keep this configurable.